pandas/tests/io/json/test_readlines.py

```diff
@@ -15,6 +15,20 @@
 from pandas.io.json._json import JsonReader
 
 
+def test_json_deprecation():
```

Comment:
```suggestion
def test_literal_json_deprecation():
```

Also can you move this test to `test_pandas.py`?